styles.css

```diff
@@ -9,11 +9,11 @@ body {
     flex-direction: column;
     align-items: center;
     position: relative;
-    width: 50%;
+    width: 30%;
     height: 400px;
     background-color: rgb(39, 38, 38);
     border: 2px solid #0f0;
```

Comment:
You make a good job so far @Albinpirela ,

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

I can suggest please remove de border